src/main/scala/com/cognite/spark/datasource/EventsRelation.scala

```diff
@@ -184,7 +204,50 @@ class EventsRelation(config: RelationConfig)(@transient val sqlContext: SQLConte
     (postUpdate, postNewItems).parMapN((_, _) => ())
   }
 
-  override val fieldsWithPushdownFilter = Seq("type", "subtype", "assetIds", "startTime", "source")
+  override val fieldsWithPushdownFilter: Seq[String] =
+    Seq("type", "subtype", "assetIds", "startTime", "source")
+
+  override def buildScan(requiredColumns: Array[String], filters: Array[Filter]): RDD[Row] = {
+    val b = urlWithBody(filters, byIdUrl)
+    val u = urlsWithFilters(filters, listUrl)
```

Comment:
This doesn't seem to be used, which seems a bit suspicious?

Reply:
removed